monitoring/files/docker-compose.yml

```diff
@@ -0,0 +1,112 @@
+version: '3.7'
+
+services:
+  grafana:
+    image: grafana/grafana:8.4.4
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "20m"
+        max-file: "10"
+    networks:
+      - monitoring
+    volumes:
+      - /etc/tvm/grafana:/var/lib/grafana
+      - /etc/tvm/grafana-provisioning/:/etc/grafana/provisioning
+      - /etc/tvm/email_template.html:/usr/share/grafana/public/emails/alert_notification.html
```

Comment:
ah is grafana configured to send alert notifications? I'm guessing recipients/alarm conditions are all configured manually through the UI?

Reply:
This is possible but not set up right now (it needs to auth to gmail or something to actually send mail)

Reply:
Yeah we can also use something like SES if we want